packages/react-focus/src/components/FocusZone/FocusZone.types.ts

```diff
@@ -163,6 +163,12 @@ export interface IFocusZoneProps extends React.HTMLAttributes<HTMLElement | Focu
    * Callback to notify creators that focus has been set on the FocusZone
    */
   onFocusNotification?: () => void;
+
+  /**
+   * If true, FocusZone prevents default behavior.
```

Comment:
It's hard as a customer to know when to use this. Is there a reason for a customer to do this or not to do this?

Reply:
I just copied the comment from their side, but I'll update it to be more descriptive.

Reply:
Once you've updated the PR to be more clear, let me know :)